octopoes/octopoes/xtdb/client.py

```diff
@@ -96,6 +97,22 @@ def get_entity(self, entity_id: str, valid_time: Optional[datetime] = None) -> d
         self._verify_response(res)
         return res.json()
 
+    def get_entity_history(
+        self,
+        entity_id: str,
+    ) -> List[TransactionRecord]:
```

Comment:
Could we also add a boundries_only argument which removes all but the first and last items from the history list?
For the reports, listing all the events, and culling them in the rocky frontend wont scale as good as doing the same in this api.